docs/storage-management.asciidoc

```diff
@@ -14,27 +14,34 @@
 [[sizing-guide]]
 == Storage and sizing guide
 
-APM Server handles a few different kinds of {apm-overview-ref-v}/apm-data-model.html[events].
-Since processing and storage costs are largely dominated by
-{apm-overview-ref-v}/transactions.html[transactions] and
-{apm-overview-ref-v}/transaction-spans.html[spans],
-only those will be covered here.
+APM processing and storage costs are largely dominated by transactions, spans, and frames.
 
-NOTE: When the sampling rate is very small, transactions will be the dominate storage cost.
+* {apm-overview-ref-v}/transactions.html[*Transactions*] describe an event captured by an Elastic APM agent instrumenting a service.
+They are the highest level of work being measuring within a service.
+* {apm-overview-ref-v}/transaction-spans.html[*Spans*] belong to transactions. They measure from the start to end of an activity,
+and contain information about a specific code path that has been executed.
+* *Frames* belong to spans. Stack frames represent a function call on the call stack,
```

Comment:
```suggestion
* *Stack frames* belong to spans. Stack frames represent a function call on the call stack,
```

IMO it's best to consistently refer to them as "stack frames" -- "frame" by itself is a bit generic. Maybe there's some precedent though?

Reply:
Makes sense. I'll update all occurrences on this page.